t/mojo/file.t

```diff
@@ -54,6 +54,10 @@ is path('file.t')->basename('.t'), basename('file.t', '.t'), 'same path';
 is path('file.t')->to_abs->dirname, scalar dirname(rel2abs 'file.t'),
   'same path';
 
+# Current file
+ok curfile->is_abs, 'path is absolute';
+is curfile, realpath(__FILE__), 'same path';
```

Comment:
I have a feeling this test might cause portability issues at some point.

Reply:
I can't think of any - realpath/abs_path only has portability issues for file paths that don't exist, and __FILE__ will always exist in this test; and curfile uses realpath itself so it will always match (the test is to ensure that it does, in fact).

Reply:
Windows Perl. 😬

Reply:
Windows is what I was referring to regarding nonexistent paths unless there's another issue I'm not aware of?